admin/src/js/services/auth.js

```diff
@@ -1,4 +1,5 @@
-const chtScriptApi = require('@medic/cht-script-api');
+const cht = require('@medic/cht-datasource');
+const chtDatasource = cht.getDatasource(cht.getRemoteDataContext());
```

Comment:
Need to decide if Sentinel should use the `remote` context and talk to api, of if we need to be passing in the `local` Pouch dbs, etc.  On one hand, we have Pouch already in context here in Sentinel, but on the other hand, we should be able to leverage the REST endpoints for all the cht-datasource interactions (since we don't have to consider offline functionality...).

Reply:
My immediate gut feeling would be to use the `remote` context, we don't need the offline-first functionality of pouch here

Reply:
After thinking about this more, I am now actually thinking that we want to use the _local_ data context in Sentinel for two reasons:

- Performance - With the remote context, every action will end up being two http "hops"; one from Sentinel to api and one from api to Couch. When using Pouch in the local context, the http calls will just go directly from Sentinel to Couch. It should not be a huge performance problem, but I cannot come up with any benefit we would get out of calling api instead of just Couch directly....
- Access - the main value of the _remote context_ is that it can function without direct access to the Couch apis.  So, if we use the remote context in the admin app and a sysadmin is running the admin app in their browser and doing stuff that uses cht-datasource, all the interactions will be going through the api server and not directly against Couch.  This supports our eventual goal of locking down the Couch apis from being directly accessed.  However, this concern does not apply to Sentinel where we can continue to support direct access to Couch since we have full control over what happens.